src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/ImageExtension.cs

```diff
@@ -50,31 +57,71 @@ public bool TryProcessAttributes(IDictionary<string, string> attributes, out Htm
                 }
             }
 
+            if(type == "complex")
+            {
+                EndingTripleColons = true;
+            } else
+            {
+                EndingTripleColons = false;
+            }
+
             if (string.IsNullOrEmpty(src))
             {
-                logError($"source is a required attribute. Please ensure you have specified a source attribute");
+                logError($"source is a required attribute. Please ensure you have specified a source attribute.");
             }
-            if (string.IsNullOrEmpty(alt))
+            if (string.IsNullOrEmpty(type))
+            {
+                logError($"type is a required attribute. Please ensure you have specified a type attribute. Options (\"comples\", \"content\", \"icon\")");
```

Comment:
Is this a typo "comples"? Should it be "complex"?

Reply:
@bharney0 - Good catch... I made a small change, and removed this error altogether. Now "content" is defaulted if no "type" attribute is present.